nwbinspector/register_checks.py

```diff
@@ -62,6 +60,8 @@ class InspectorMessage:
         The name of the instantiated object being inspected.
     location : str
         The location relative to the root of the NWBFile where the inspected object may be found.
+    file : str
+        The path to the NWBFile this message pertains to.
```

Comment:
Upon using this code for the past day, I actually think there is sufficient identifiability just from the filename alone. 

Two questions then,

(a) Should this attribute be renamed to (i) `filename`, (ii) `file_name`, (iii) or `file_path` to make it clear it's not an open `file` object (one of the other common uses of that keyword)?

(b) 
```suggestion
    file : str
        The name of the NWBFile this message pertains to.
```

Reply:
@bendichter Thoughts on this or good as is?

Reply:
"filename" or "file_name" would be fine. Let's not call it a path unless it is truly a path. I think we should keep the .nwb

Reply:
A quick glance around other Python references, seems `filename` is the most common, so going with that.